packages/contracts-bedrock/scripts/deploy/ChainAssertions.sol

```diff
@@ -50,7 +51,14 @@ library ChainAssertions {
     }
 
     /// @notice Asserts that the SystemConfig is setup correctly
-    function checkSystemConfig(Types.ContractSet memory _contracts, DeployConfig _cfg, bool _isProxy) internal view {
+    function checkSystemConfig(
+        Types.ContractSet memory _contracts,
```

Comment:
Since we've introduced `DeployOPChainInput` here, I think we can also get rid of `ContractSet` and replace it with `DeployOPChainOutput`.